### PR TITLE
Updating retry, bundling settings

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**
This corresponds to recommendations made by the Google Cloud Pub/Sub team.

### CODE REVIEW

File: google/pubsub/v1/pubsub_gapic.yaml [hunk 1]
```diff
@@ -168,17 +176,29 @@ interfaces:
     retry_codes:
     - UNAVAILABLE
     - DEADLINE_EXCEEDED
+  - name: one_plus_delivery
```
Comment (shinfan) [inline google/pubsub/v1/pubsub_gapic.yaml]: Why is this one_plus_delivery identical to idempotent?

Comment (garrettjonesgoogle) [inline google/pubsub/v1/pubsub_gapic.yaml]: It happens to be the same now, but I'm thinking that these two cases might differ in the future.

File: google/pubsub/v1/pubsub_gapic.yaml [hunk 2]
```diff
@@ -32,12 +32,20 @@ interfaces:
...
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000 # 60 seconds
+    initial_rpc_timeout_millis: 12000 # 12 seconds
+    rpc_timeout_multiplier: 1
+    max_rpc_timeout_millis: 12000 # 12 seconds
+    total_timeout_millis: 600000 # 10 minutes
+  - name: messaging
+    initial_retry_delay_millis: 100
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000 # 60 seconds
```
Comment (michaelbausor) [inline google/pubsub/v1/pubsub_gapic.yaml]: In the doc comments, Emilio and Haakon suggest a lower timeout for Publish/Pull/Ack (10s)

Comment (garrettjonesgoogle) [inline google/pubsub/v1/pubsub_gapic.yaml]: Oops, I swapped the two definitions. I will fix that.

**Comment by michaelbausor** [comment]
LGTM

**Pull request merged**